torchvision/io/__init__.py

```diff
@@ -98,15 +98,15 @@ class VideoReader:
             Currently available options include ``['video', 'audio']``
     """
 
-    def __init__(self, path, stream="video"):
+    def __init__(self, path, stream="video", num_threads=0):
```

Comment:
Worth adding the new argument in the pydoc.

Reply:
Yeah, makes sense. Pushed the changes